.github/pull_request_template.md

```diff
@@ -1,69 +1,78 @@
+* * *
 
-New or updated [recipe/diagnostic](https://docs.esmvaltool.org/en/latest/community/diagnostic.html):
+## Before you get started
 
-- [ ] Documentation for the recipe/diagnostic is available in the `doc/sphinx/source/recipes` folder and an entry has been added to `index.rst`
-- [ ] [Provenance information](https://docs.esmvaltool.org/en/latest/community/diagnostic.html#recording-provenance) has been added and no warnings related to provenance are generated when running the recipe
+<!--
+    Please discuss your idea with the development team before getting started,
+    to avoid disappointment or unnecessary work later. The way to do this is
+    to open a new issue on GitHub.
+-->
 
-New or updated [data reformatting script](https://docs.esmvaltool.org/en/latest/develop/dataset.html):
+- [ ] [☝ Create an issue](https://github.com/ESMValGroup/ESMValTool/issues) to discuss what you are going to do
 
-- [ ] Add a new dataset to the table in the [documentation](https://docs.esmvaltool.org/en/latest/input.html#supported-datasets)
-- [ ] Add a test for the CMORized data to recipes/example/recipe_check_obs.yml and run the recipe, to make sure the CMOR checks pass without errors
-- [ ] There are clear instructions on how to obtain the data
-- [ ] Tag @remi-kazeroni in this pull request, so that the new dataset can be added to the OBS data pool at DKRZ and synchronized with CEDA-Jasmin
+## Checklist
 
-Automated checks pass, status can be seen below the pull request:
+It is the responsibility of the author to make sure the PR is ready to review. The icons indicate whether the item will be subject to the [🛠 Technical][1] or [🧪 Scientific][2] review.
 
-- [ ] Circle/CI tests pass. If the tests are failing, click the `Details` link to find out why.
-- [ ] Preferably Codacy code quality checks pass, however a few remaining hard to solve Codacy issues are still acceptable. If there is an error, click the link to find out why. If you suspect Codacy may be wrong, please ask by commenting.
-- [ ] The documentation is building successfully on readthedocs and looks well formatted, click the `Details` link to see it.
+- [ ] [🛠][1] PR has a descriptive title for the [changelog](https://docs.esmvaltool.org/en/latest/community/introduction.html#branches-pull-requests-and-code-review)
+- [ ] [🛠][1] Code follows the [style guide](https://docs.esmvaltool.org/en/latest/community/introduction.html#code-style)
+- [ ] [🛠][1] [Documentation](https://docs.esmvaltool.org/en/latest/community/introduction.html#documentation) is available
+- [ ] [🛠][1] YAML files pass [`pre-commit`](https://docs.esmvaltool.org/en/latest/community/introduction.html#pre-commit) or [`yamllint`](https://docs.esmvaltool.org/en/latest/community/introduction.html#yaml) checks
+- [ ] [🛠][1] [Circle/CI tests pass](https://docs.esmvaltool.org/en/latest/community/introduction.html#branches-pull-requests-and-code-review)
```

Comment:
Maybe link to https://esmvaltool--1924.org.readthedocs.build/en/1924/community/review.html#check-automated-tests instead?

Reply:
We thought about that, but that really is the checklist for the review. @nielsdrost and I decided that it is better to link to the more extensive description in the contribution guidelines and in this case there is a relevant section.

Reply:
Maybe I'm missing something here, but I think the checklist in the pull request template should be aligned with the checklist on the review page, with short entries in the pull request template and longer explanations on the review page. If needed, we can then link from the review page to other places in the documentation that have even more information.

Reply:
The review page is a simple checklist for things to look at. They say nothing about how to actually achieve such check. In that case, linking to the contributing guidelines directly makes more sense, and which is what we did.

Reply:
I like the links to the contributing guidelines. I understand the reasoning that you would want to link to the review page, but with the link to the more extensive descriptions it speeds things up for the reviewers if they have to actually look things up since they do not have to follow two links (first to the review page and then to the contributing guidelines).

Reply:
Ideally the review checklist and contribution guidelines would match up in structure, so the section headings would _be_ the checklist. We propose to let this be part of a followup PR.